schemas/comorbidity.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "comorbidity",
+  "description": "The collection of data elements related to a donor's comorbidities",
```

Comment:
I think we should define comorbidies in this description, and emphasize the multiple submissions like the other tables: 

```The collection of data elements related to a donor's comorbidities. Comorbidities are any distinct entity that has existed or may occur during the clinical course of a patient who has the index disease under study. The primary diagnosis is the first diagnosed case of cancer in a donor. To submit multiple comorbidities for a single donor, submit multiple rows in the comorbidity file for this donor.```

Reference: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2713155/

Reply:
Updated description as suggested

Reply:
@hknahal i made a copy pasta mistake in this description and it still has a primary diagnosis note.  I've fixed that below: 
```
The collection of data elements related to a donor's comorbidities. Comorbidities are any distinct entity that has existed or may occur during the clinical course of a patient who has the index disease under study. To submit multiple comorbidities for a single donor, submit multiple rows in the comorbidity file for this donor
```

Reply:
@rosibaj I updated the description to the corrected one.